stbt_auto_selftest.py

```diff
@@ -77,7 +80,6 @@ def main(argv):
     subparsers.add_parser('validate', help='Run (and check) the auto-selftests')
 
     cmdline_args = parser.parse_args(argv[1:])
-    screenshots_root = cmdline_args.screenshots_root or "selftest/screenshots"
```

Comment:
So, to be clear, my commit is a refactoring that doesn't change the existing behaviour, but now you think the behaviour should be different? I'll add an item to the todo list.